modules/ROOT/pages/reliability-patterns.adoc

```diff
@@ -0,0 +1,164 @@
+= Reliability Patterns
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: reliability, patterns, cluster, ha, transaction, delivery, ensure
+
+A reliability pattern is a design that results in reliable messaging for an application, even if the application receives messages from a non-transactional transport. A reliability pattern couples a reliable acquisition flow with an application logic flow, as shown in the following diagram:
+
+image::reliability-pattern.png[Reliability_Pattern]
+
+The reliable acquisition flow (the left side of the diagram) delivers a message reliably from a message source that does not implement transactions to an outbound operation of a connector that implements transactions. The operation can be of any type of the transactional endpoints, such as VM or JMS. If the reliable acquisition flow cannot deliver the message, it ensures that the message isn't lost.
+
+* For socket-based connections like HTTP, this means returning an "unsuccessful request" response to the client so that the client can retry the request.
```

Comment:
...this means returning an "unsuccessful request" response so the client can retry the request...
remove _to the client_ so _that_ the client